### ISSUE TITLE
BioPAX processor warning messages to look into

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#129]
We occasionally get warnings like this:
```
WARNING: indra/uniprot - Could not find protein with id P
WARNING: indra/uniprot - Could not find protein with id Q
```
This could be due to incorrectly extracting some protein IDs from the BioPAX model.

**Comment by bgyori** [comment]
```
INFO: indra/biopax - Unknown modification type term: hydroxylated proline
```
This one should be handled as an Agent ModCondition.

**Comment by bgyori** [comment]
First one solved in 55f20e1d992962f57743e0c93fd95c82249dd2fe

**Comment by bgyori** [comment]
Second one covered in daa93cf017b2fa81db0e0c7ab2f5216ac6ebf8c9
